### ISSUE TITLE
Fix errors while leaving group

### ISSUE THREAD

**Issue by Silver-IT** [okTurtles/group-income#1839]
### Problem

**Steps to reproduce**
- [Tab1] `u1` signs up, and creates a group
- [Tab2] `u2` joins the group
- [Tab2] `u2` leaves the group by himself
- [Tab1] have two errors in console
![image](https://github.com/okTurtles/group-income/assets/57976479/0b8c7503-538d-46c5-a92d-82b3e9098ac1)
![image](https://github.com/okTurtles/group-income/assets/57976479/ef5bcf1a-afc4-42d0-8164-e696d6de09a5)
- [Tab2] have two errors in console
![image](https://github.com/okTurtles/group-income/assets/57976479/4a6e0d84-bba9-4676-a532-04791594f50c)
![image](https://github.com/okTurtles/group-income/assets/57976479/123582d6-134c-4811-b2b6-418180a784cb)

### Solution

Fix all the errors in console

**Comment by Silver-IT** [comment]
Additionally, I can see more errors with the following steps.

- [Tab1] `u1` signs up, and creates group (doesn't log out)
- [Tab2] `u2` joins the group (doesn't log out)
- [Tab3] `u2` logs in
- [Tab3] `u2` leaves the group
- [Tab1] there are 3 errors
- [Tab2] there are 6 errors
- [Tab3] there are 2 errors

**Comment by Silver-IT** [comment]
Also, if `u3` is leaved by proposal, there are more errors.

- [Tab1] `u1` signs up, and creates group
- [Tab2] `u2` joins the group
- [Tab3] `u3` joins the group
- [Tab2] `u2` creates a proposal to kick `u3`
- [Tab1] `u1` approves the proposal, and kicks `u3`
- [Tab1] there are 4 errors
- [Tab2] there are 2 errors

**Comment by Silver-IT** [comment]
@corrideat, based on your detailed explanation [here](z9brRu3VJnGhD792p8WTCUSQbmRdUQGk57nxkmnbc1KC1sdoghGn), I checked the steps several times. And I found that they are not errors of duplicated events, are of simulated processMessage function call before publishEvent. So that's what we can ignore at the moment.

I will close this issue now, but however we should create another one in which we can work for those kinds of errors to be handled in a different way.
